app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/MockService.java

```diff
@@ -0,0 +1,16 @@
+package com.greenfox.peridot.peridot_coz_android.model.api;
+
+import com.greenfox.peridot.peridot_coz_android.model.pojo.User;
+
+import retrofit2.Call;
+
+public class MockService implements UserInterface {
+
+    public MockService() {
+    }
+
+    @Override
+    public Call<User> getUser() {
+        return null;
```

Comment:
we thought retrofit 2.0 would only work if we packed User into a Call, as per the Dev's video ( https://www.youtube.com/watch?v=t34AQlblSeE ), now I realize its not necessary, will be fixed in next commit